### ISSUE TITLE
dip fails with psych 4.0.0

### ISSUE THREAD

**Issue by hachi8833** [bibendi/dip#129]
**Describe the bug**

`dip` fails with the breaking changes in the newer psych 4.0.0 gem:

https://github.com/ruby/psych/pull/487

**To Reproduce**

1. `gem update psych`
2. run any dip commands such as `dip ls`.
3. dip fails.

```sh
$ dip sh
/Users/<username>/.config/anyenv/envs/rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/psych-4.0.0/lib/psych.rb:322:in `safe_load': wrong number of arguments (given 4, expected 1) (ArgumentError)
	from /Users/<username>/.config/anyenv/envs/rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/dip-7.1.0/lib/dip/config.rb:65:in `load_yaml'
	from /Users/<username>/.config/anyenv/envs/rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/dip-7.1.0/lib/dip/config.rb:107:in `config'
	from /Users/<username>/.config/anyenv/envs/rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/dip-7.1.0/lib/dip/config.rb:90:in `block (2 levels) in <class:Config>'
	from /Users/<username>/.config/anyenv/envs/rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/dip-7.1.0/lib/dip/cli.rb:27:in `start'
	from /Users/<username>/.config/anyenv/envs/rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/dip-7.1.0/exe/dip:23:in `<top (required)>'
	from /Users/<username>/.anyenv/envs/rbenv/versions/3.0.1/bin/dip:23:in `load'
	from /Users/<username>/.anyenv/envs/rbenv/versions/3.0.1/bin/dip:23:in `<main>'
```

The symptom is not related with the configs in dip.yml or docker-compose.yml.

**Expected behavior**

Any `dip` commands should work with psych 4.0.0.

Actually, just running `gem uninstall psych` resolves the issue because psych 3.3.0 is a "default gem" in the current Ruby 3.0.1p64. 

```sh
$ gem uninstall psych
$ gem list|grep psych
psych (default: 3.3.0)
```

**Context (please complete the following information):**
 - OS: macOS Big Sur 11.3.1
 - Ruby version: 3.0.1p64
 - Dip Version 7.1.0

**Additional context**

I guess there might occur some more fixes or changes regarding psych breaking changes. The situation might be in fluid.

https://bugs.ruby-lang.org/issues/17866

Rails has also been affected by the changes in psych:

https://github.com/rails/rails/pull/42249

**Comment by bibendi** [comment]
Sorry for the long delay. I've been on vacation.
I will try to resolve the bug in this or next week.

**Comment by alec-c4** [comment]
hey, any news regarding this bug?

**Comment by bibendi** [comment]
Hi,
Fixed in v7.1.1

**Comment by hachi8833** [comment]
Thank you very much!
